docs/coordinates/frames.rst

```diff
@@ -288,12 +288,12 @@ Transforming between Frames
 
 To transform a frame object with data into another frame, use the
 ``transform_to`` method of an object, and provide it the frame you wish to
-transform to. This frame can either be a frame *class*, in which case
-the default attributes will be used, or a frame object (with or without
-data)::
+transform to.  This frame should be a frame object (with or without data).
```

Comment:
Just to be explicit here for people who may have missed or skipped earlier discussion, maybe
```suggestion
transform to.  This frame should be a frame object (with or without coordinate data).
```

Reply:
Done